### PR TITLE
Add archive timeout override

### PR DESCRIPTION
**Pull request by ericvaandering**

### CODE REVIEW

File: charts/rucio-daemons/templates/conveyor-submitter.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ spec:
...
...orTransferSubmitter.sourceStrategy }}{{ end }} {{- if .Values.conveyorTransferSubmitter.excludeRses }} --exclude-rses {{ .Values.conveyorTransferSubmitter.excludeRses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.includeRses }} --include-rses {{ .Values.conveyorTransferSubmitter.includeRses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.rses }} --rses {{ .Values.conveyorTransferSubmitter.rses }}{{ end}} {{- if .Values.conveyorTransferSubmitter.sleepTime }} --sleep-time {{ .Values.conveyorTransferSubmitter.sleepTime }}{{ end}} {{- if .Values.conveyorTransferSubmitter.activities }} --activities {{ .Values.conveyorTransferSubmitter.activities }}{{ end}} {{- if .Values.conveyorTransferSubmitter.excludeActivities }} --exclude-activities {{ .Values.conveyorTransferSubmitter.excludeActivities }}{{ end}} {{- if .Values.conveyorTransferSubmitter.maxSources }} --max-sources {{ .Values.conveyorTransferSubmitter.maxSources }}{{end }} {{- if .Values.conveyorTransferSubmitter.retryOtherFts }} --retry-other-fts {{- end}} {{- if kindIs "float64" .Values.conveyorTransferSubmitter.archiveTimeout }}  --archive-timeout-override {{ .Values.conveyorTransferSubmitter.archiveTimeout }}{{ end}}"
```
Comment (rcarpa) [inline charts/rucio-daemons/templates/conveyor-submitter.yaml:104]: --retry-other-fts {{- end}}

Optional change: I would remove the "-" in {{- end}} ; it is the only end which strips the white spaces; not problematic now; but could be if somebody adds another option and also strips the white spaces

**Review by rcarpa — APPROVED** [review]

**Comment by rcarpa** [comment]
To be noted that retry-other-fts  is scheduled to be removed soon : https://github.com/rcarpa/rucio/commit/9b1119924140c2c19b0838aeb51089b9c652e817

File: charts/rucio-daemons/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: rucio-daemons
-version: 1.26.0
+version: 1.26.1
```
Comment (bari12) [inline charts/rucio-daemons/Chart.yaml:2]: I just release `1.26.1` so this needs to be `1.26.2` then.

Comment (ericvaandering) [reply charts/rucio-daemons/Chart.yaml]: Done

**Comment by ericvaandering** [comment]
All done. Please merge

**Review by rcarpa — APPROVED** [review]

**Pull request merged**